common/browsered/field-validation-checks.js

```diff
@@ -51,15 +53,31 @@ exports.isPhoneNumber = function (value) {
 
 exports.isHttps = function (value) {
   if (value.substr(0, 8) !== 'https://') {
-    return validationErrors.isHttps
+    return validationErrors.https
   } else {
     return false
   }
 }
 
 exports.isBelowMaxAmount = value => {
   if (!exports.isCurrency(value) && parseFloat(value) >= MAX_AMOUNT) {
-    return validationErrors.isBelowMaxAmount
+    return validationErrors.belowMaxAmount
   }
   return false
 }
+
+exports.isSortCode = value => {
+  if (!/^\s?(\d{2}\s?-?){2}\d{2}\s?$/.test(value)) {
```

Comment:
Some corrections:
`12 -12 -12` - matches, ok
`1212-12` - matches, ok (I suppose or we can check explicitly for "-" and use two types of regex (one with whitespace only and one combined with "-"), but maybe it's too complicated), let me know what you think
`12 - 12 - 12` - doesn't match
maybe it's better: `^\s?(\d{2}\s?-?\s?){2}\d{2}\s?$` (fixes the `12 - 12 - 12`)

Reply:
The conventions tend to be:
dash separated: `12-12-12`
space separated: `12 12 12`
or just: `121212`

Its unlikely someone would do 1212-12 but I think its fine to match it if they did.

`12 -12 -12` is weird as its a mix but I suppose it could happen, so I don’t see the harm in making `12 - 12 - 12` work

Reply:
We will need some unit tests for that as well (probably we should add tests for all of the validators as there are no tests at all), covering the rest of the functions is not part of this PR, but it's good to have, let me know what you think

Reply:
@jonheslop yes, if you decide we can have two types of regex to remove the weird `1212-12` as I mentioned above

Reply:
Yep should write tests.

I dont think we need another regex, we should be flexible so although `1212-12` is weird we should allow it anyway